third_party/move/testing-infra/transactional-test-runner/src/vm_test_harness.rs

```diff
@@ -326,20 +311,16 @@ impl<'a> MoveTestAdapter<'a> for SimpleVMTestAdapter<'a> {
         let traversal_storage = TraversalStorage::new();
 
         let serialized_return_values = self
-            .perform_session_action(
-                gas_budget,
-                |session, gas_status| {
-                    session.execute_function_bypass_visibility(
-                        module,
-                        function,
-                        type_args,
-                        args,
-                        gas_status,
-                        &mut TraversalContext::new(&traversal_storage),
-                    )
-                },
-                VMConfig::from(extra_args),
-            )
+            .perform_session_action(gas_budget, |session, gas_status| {
+                session.execute_function_bypass_visibility(
+                    module,
+                    function,
+                    type_args,
+                    args,
+                    gas_status,
+                    &mut TraversalContext::new(&traversal_storage),
+                )
+            })
```

Comment:
here and above, are extra_args even needed anymore? only seems used in logging now?

Reply:
This is great extensibility to add adapter args here haha. I guess we still need them to pass `--verbose`?